.github/CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# BDK Generator Contribution and Governance Policies
+# Symphony BDK Generator Contribution and Governance Policies
```

Comment:
Should we just name it Symphony Generator (as it also contains the WDK)?

Reply:
that was bugging me. Let's do that.